pkg/apis/compliance/v1alpha1/complianceremediationcheck_types.go

```diff
@@ -0,0 +1,57 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ComplianceCheckStatus string
+
+const (
+	// The check ran to completion and passed
+	CheckResultPass ComplianceCheckStatus = "PASS"
+	// The check ran to completion and failed
+	CheckResultFail ComplianceCheckStatus = "FAIL"
+	// The check ran to completion and found something not severe enough to be considered error
+	CheckResultInfo ComplianceCheckStatus = "INFO"
+	// The check ran, but could not complete properly
+	CheckResultError ComplianceCheckStatus = "ERROR"
+	// The check didn't run because it is not applicable or not selected
+	CheckResultSkipped ComplianceCheckStatus = "SKIP"
+	// The check didn't yield a usable result
+	CheckResultNoResult ComplianceCheckStatus = ""
+)
+
+// ComplianceCheckResultSpec defines the desired state of ComplianceCheckResult
+type ComplianceCheckResultSpec struct {
+	// A unique identifier of a check
+	ID string `json:"id"`
+	// The result of a check
+	Result ComplianceCheckStatus `json:"result"`
```

Comment:
Do we want to change this to be 'status' as well?

Reply:
Yes, thanks, fixed locally.